.editorconfig

```diff
@@ -40,4 +40,6 @@ charset = utf-8
 # Automake files. TODO: Should this do indent_style = tab ?
```

Comment:
If we're changing it to tab style (which is reasonable, because Makefile's are tab specified), two things should change here, and a suggestion for improvements:

1) Remove the TODO: above :-)

2) Make sure this is the expectations in Makefile.am's ? Remember, the Makefile's are actually generated via first a Makefile.in, and then to Makefile.

3) Possibly: If tabs are the right thing to do in Makefile.am's, we should also make sure to fix all Makefile.am's that might not be using <tab> when they should. That can be a separate Issue / Jira, but something to keep in mind.

Reply:
> 1. Remove the TODO: above :-)

Done.

> 2. Make sure this is the expectations in Makefile.am's ? Remember, the Makefile's are actually generated via first a Makefile.in, and then to Makefile.

automake documentation references make, and make recipies need to be intented with tab characters and not spaces. However...

> 3. Possibly: If tabs are the right thing to do in Makefile.am's, we should also make sure to fix all Makefile.am's that might not be using when they should. That can be a separate Issue / Jira, but something to keep in mind.


... I did normalize most of the white-space in our existing *.am files, and I found that for variable assignment/definitions, leading tabs result in missing the automake variable definitions while leading spaces are OK. This means I had to "out-dent" the variable assignment statements after normalizing spaces to tabs in a few files.

According to [automake docs](https://www.gnu.org/software/automake/manual/automake.html#GNU-Build-System):

> Generally, Automake is not particularly smart in the parsing of unusual Makefile constructs, so you’re advised to avoid fancy constructs or “creative” use of whitespace. For example, TAB characters cannot be used between a target name and the following “:” character, and variable assignments shouldn’t be indented with TAB characters.

This could be one such instance, where we really should not have indented variable assignments to begin with, but since we did, we cannot convert those leading spaces to tabs and expect it to parse correctly.
I have updated the branch after having let it configure and build over lunch without issues.